db/migrate/20160329163708_add_carrier_and_service_level_to_spree_shipping_methods.rb

```diff
@@ -0,0 +1,6 @@
+class AddCarrierAndServiceLevelToSpreeShippingMethods < ActiveRecord::Migration
+  def change
+    add_column :spree_shipping_methods, :carrier, :string
```

Comment:
Are there template overrides and additional permitted attributes to support these fields to the shipping-method backoffice?